src/main/java/it/gov/pagopa/reward/service/reward/evaluate/UserInitiativeCountersUpdateServiceImpl.java

```diff
@@ -83,6 +84,8 @@ public Mono<RewardTransactionDTO> update(UserInitiativeCountersWrapper userIniti
 
                                         /* set RewardCounters in RewardTransactionDTO object */
                                         reward.setCounters(rewardCountersMapper.apply(initiativeCounter, ruleEngineResult, initiativeConfig));
+                                        reward.setFamilyId(InitiativeGeneralDTO.BeneficiaryTypeEnum.NF.equals(initiativeConfig.getBeneficiaryType())
+                                        ? initiativeCounter.getUserId() : null);
```

Comment:
move this line on an external method, use "if" instead of the ternary operator in order to set the field only if required (it will be already null)